src/components/form-editor/form-editor.component.tsx

```diff
@@ -39,6 +39,17 @@ interface ErrorProps {
   title: string;
 }
 
+interface markerProps {
+  startCol: number;
+  startRow: number;
+  endCol: number;
+  endRow: number;
+  className: string;
+  // eslint-disable-next-line @typescript-eslint/no-redundant-type-constituents, @typescript-eslint/no-explicit-any
+  type: string | any;
```

Comment:
```suggestion
  type: string | unknown;
```

Does `unknown` work here?

Reply:
unknown doesn't work, I had used it at first but it was failing.